lib/notifier.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+class Notifier
+  def initialize(config_loader_klass:)
+    config_loader = config_loader_klass.new('./support/config.yaml')
+    account_sid = config_loader.account_sid
+    auth_token = config_loader.auth_token
+    @client = Twilio::REST::Client.new account_sid, auth_token
+  end
+  def notify(estimated_time)
+    time = "#{estimated_time.hour}:#{estimated_time.min}"
+    body = "Thank you! Your order was placed and will be delivered before #{time}"
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)